.github/workflows/Procedure_push_docker_images.yml

```diff
@@ -109,6 +149,6 @@ jobs:
         IFS=',' read -r -a images <<< "$IMAGE_NAMES"
         for image in "${images[@]}"; do
           echo "Tagging and pushing wazuh/$image:${WAZUH_VERSION} to wazuh/$image:$IMAGE_TAG"
-          docker tag wazuh/$image:${WAZUH_VERSION} wazuh/$image:$IMAGE_TAG
-          docker push wazuh/$image:$IMAGE_TAG
+          docker tag wazuh/$image:${WAZUH_VERSION} ${{ secrets.DOCKERHUB_USERNAME }}/$image:$IMAGE_TAG
```

Comment:
Please, restore these changes: replace ${{ secrets.DOCKERHUB_USERNAME }} by `wazuh` in lines 152 and 153.

Reply:
Done ✅